frontend/src/libs/hooks/use-media-query/use-media-query.hook.ts

```diff
@@ -0,0 +1,27 @@
+import { useCallback, useEffect, useState } from '../hooks.js';
+
+const useMediaQuery = (query: string): boolean => {
+  const [isMatched, setIsMatched] = useState(false);
+
+  const handleMediaQueryChange = useCallback(
+    (event: MediaQueryListEvent): void => {
+      setIsMatched(event.matches);
+    },
+    [],
```

Comment:
Is it acceptable to have no dependencies there even without `setIsMatched`?

Reply:
I guess React knows that `setIsMatched` would never be recreated and doesn't ask us to add this function in the dependency array